doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -395,7 +395,7 @@ stands for :n:`let @ident := fun {+ @binder} => @term in @term’`.
 Definition by case analysis
 ---------------------------
 
-Objects of inductive types can be destructurated by a case-analysis
+Objects of inductive types can be destructed by a case-analysis
```

Comment:
Not sure if this should be "destructed" or "destructured".

Reply:
Good question. Maybe "destructured" would be more appropriate indeed.